contracts/AnyswapV4CallProxy.sol

```diff
@@ -110,44 +110,139 @@ abstract contract Whitelistable is MPCManageable {
     }
 }
 
-abstract contract Billable is Whitelistable {
-    event Fund(address indexed beneficiary, uint256 amount);
+interface IERC20 {
```

Comment:
we may need use `SafeERC20` library to process ERC20 token transfers, as some ERC20 token's implementation does not have `returns (bool)` in   their `transfer` or `transferFrom` method. (eg. [Tether: USDT](https://cn.etherscan.com/address/0xdac17f958d2ee523a2206206994597c13d831ec7#code))